validator/src/main/java/com/google/daq/mqtt/sequencer/SequenceBase.java

```diff
@@ -2203,7 +2203,6 @@ protected int countReceivedEvents(Class<?> clazz) {
 
   protected <T> List<T> popReceivedEvents(Class<T> clazz) {
     SubFolder subFolder = CLASS_SUBFOLDER_MAP.get(clazz);
-    safeSleep(EVENT_WAIT_DELAY_MS);
```

Comment:
hmmm, not sure we can do this. The problem is that there's can be a long delay from when events are sent and ultimately received -- and there's no (easy) deterministic way to know it's been "long enough" ... Is there a particular problem this was causing?

Reply:
When 50 devices try to send pointset and system events, a 100-second random delay makes it impossible to run the sequencer for proxy devices.

Reply:
As discussed -- we'll leave this for now and then clean it up in a later pass (bug filed).